spec/features/user_post_index_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+describe '#' do
```

Comment:
#### OPTIONAL

- [ ] In your describe text  you can write 'posts #index'  of  only #, this will make your code more readable  👍🏿